src/internal/config.js

```diff
@@ -137,6 +138,7 @@ module.exports = {
   featureToggles: {
     manageAgreements: ['local', 'dev', 'development', 'test', 'preprod'].includes(process.env.NODE_ENV),
     chargeInformation: ['local', 'dev', 'development', 'test', 'preprod'].includes(process.env.NODE_ENV),
-    manageInvoiceAccounts: ['local', 'dev', 'development', 'test', 'preprod'].includes(process.env.NODE_ENV)
+    manageInvoiceAccounts: ['local', 'dev', 'development', 'test', 'preprod'].includes(process.env.NODE_ENV),
+    deleteAllBillingData: !isProductionLike
```

Comment:
I have no real issue with this, but 

```
['local', 'dev', 'development', 'test'].includes(process.env.NODE_ENV)
```

...would probably fit the convention better.

Reply:
Fixed